changelog/1633.removal.1.rst

```diff
@@ -0,0 +1,3 @@
+The ``integer_charge`` attributes of |Particle| and |IonicLevel| have
+been removed after being deprecated in ``v0.7.0``. Use the
+``charge_number`` attribute instead.
```

Comment:
I think I see your logic behind the split of these changelogs into three different notes, but just for future reference, if anyone's removing related, deprecated functionality, would they be expected to also do this, well, dry and formal split, or is a single note fine?

Reply:
Good question!  I'd say it would be fine to do it either way, so long as the changelog entry doesn't end up too bulky.  My usual strategy is to have each changelog describe one change, though what "one change" means can be a bit nebulous.  For this PR, I split them up because it was easier to write that way since there were a couple of small differences between the different changes.

Reply:
Makes sense! 👍🏻